src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/DirectiveAttributeCompletionItemProvider.cs

```diff
@@ -142,13 +163,40 @@ internal static ImmutableArray<RazorCompletionItem> GetAttributeCompletions(
                 displayText,
                 insertText,
                 descriptionInfo: new([.. attributeDescriptions]),
-                commitCharacters: razorCommitCharacters.ToImmutableAndClear());
+                commitCharacters: razorCommitCharacters.ToImmutableAndClear(),
+                isSnippet);
 
             completionItems.Add(razorCompletionItem);
         }
 
         return completionItems.ToImmutableAndClear();
 
+        bool TryGetSnippetText(
```

Comment:
>bool TryGetSnippetText( [](http://example.com/codeflow?start=8&length=23)

nit: can this be static instead to be more explicit about what is being captured?